app/views/home/_swaps_open.html.haml

```diff
@@ -39,16 +39,17 @@
     preferred_party_id = $("select[name='user[preferred_party_id]']").val();
     willing_party_id = $("select[name='user[willing_party_id]']").val();
     if (preferred_party_id !== null && willing_party_id !== null) {
-      showModal($('.js-login-modal'));
+      $("#js-login-modal").modal();
     } else {
-      showModal($('.js-error-modal'));
+      $("#js-error-modal").modal();
     }
   }
 
-.smv-modal.js-error-modal{ style: "display: none;" }
-  .smv-modal-dialog
-    %i.fa.fa-times.subdued.modal-close
-    %p.text-center
-      Please choose both your preferred party and your willing party.
-    %p.text-center
-      %a.btn.btn-primary.modal-close{ href: "#" } OK
+.modal#js-error-modal{ tabindex: "-1", role: "dialog" }
```

Comment:
Same change in a few places, we use a BS modal which requires some extra content (`.modal-content`).